tests/TagTest.php

```diff
@@ -43,6 +43,26 @@ public function it_automatically_generates_a_slug()
     }
 
     /** @test */
+    public function it_uses_str_slug_if_config_slugger_value_is_empty()
+    {
+        app('config')->set('laravel-tags.slugger', '');
+
+        $tag = Tag::findOrCreateFromString('this is a tag');
+
+        $this->assertSame('this-is-a-tag', $tag->slug);
+    }
+
+    /** @test */
+    public function it_uses_the_specified_method_defiend_in_config_slugger_to_generate_a_slug()
+    {
+        app('config')->set('laravel-tags.slugger', 'str_slug');
```

Comment:
Use another function that produces a different result than the `str_slug` function. Perhaps `strtoupper`? In that way we know for certain that the custom function is being used.